lib/plug/parsers.ex

```diff
@@ -193,7 +193,8 @@ defmodule Plug.Parsers do
       {"content-type", ct} ->
         case Conn.Utils.content_type(ct) do
           {:ok, type, subtype, headers} ->
-            reduce(conn, Keyword.fetch!(opts, :parsers), type, subtype, headers, opts)
+            content_type = %{type: type, subtype: subtype, headers: headers}
```

Comment:
For a moment I was wondering what a bad name `content_type` for a map containing `headers`, but after checking `Conn.Utils.content_type/3` it turns out that third thing returned is `params`, not `headers`. Let's use that then.